src/css/style.css

```diff
@@ -49,116 +49,54 @@ button {
 }
 
 .game-canvas {
+    display: flex;
+    flex-direction: column;
```

Comment:
This way we don't need all the specific classes further down `.cell-8` , `cell-10`, ... We set the max width and height we want to see and flex-box does the rest.